src/main/java/com/google/api/codegen/discovery/transformer/php/PhpSampleMethodToViewTransformer.java

```diff
@@ -169,6 +169,8 @@ private SamplePageStreamingView createSamplePageStreamingView(
     builder.isResourceMap(fieldInfo.type().isMap());
     builder.pageVarName(
         symbolTable.getNewSymbol(namer.localVarName(Name.lowerCamel(fieldInfo.name()))));
+    builder.pageTokenName(methodInfo.requestPageTokenName());
```

Comment:
How come we need case manipulation for the nextPageToken and  not for pageToken?